README.md

```diff
@@ -1,6 +1,7 @@
 # WorkOS .NET Library
 
-[![Build Status](https://workos.semaphoreci.com/badges/workos-dotnet/branches/main.svg?style=shields&key=343c1d18-79da-4ea3-89ce-8a6195a9d3d9)](https://workos.semaphoreci.com/projects/workos-dotnet)
+[![Packagist Version](https://img.shields.io/nuget/v/WorkOS.net)](https://www.nuget.org/packages/WorkOS.net)
```

Comment:
This should be a link to the package. In most of our repos it's just a link to the image (see [workos-php](https://github.com/workos/workos-php/))

I'll go through and fix those at some point.